pages/common/wlc.md

```diff
@@ -0,0 +1,24 @@
+# wlc
+
+> Weblate <https://weblate.org/> command-line utility.
```

Comment:
```suggestion
> Weblate utility.
```
The More information line already exists. Do not use webpages here.
Furthermore, try not to use command-line in the description as it's always implied. 
Expand on this description about what you actually do with the utility.